src/main/java/com/powsybl/openloadflow/util/Reports.java

```diff
@@ -195,4 +196,86 @@ public static Reporter createPostContingencySimulation(Reporter reporter, String
         return reporter.createSubReporter("postContingencySimulation", "Post-contingency simulation '${contingencyId}'",
                 "contingencyId", contingencyId);
     }
+
+    public static Reporter createNewtonRaphsonReporter(Reporter reporter, boolean detailedNrLogs, int networkNumCc, int networkNumSc, int outerLoopIteration, String outerLoopType) {
+        if (detailedNrLogs) {
+            if (outerLoopIteration == 0) {
```

Comment:
As previously explained, there is something badly designed using this info here.